lang/en/texts/sendinblue.html

```diff
@@ -169,4 +169,4 @@
 </script>
 <script src="https://www.google.com/recaptcha/api.js?hl=fr"></script>
```

Comment:
We may not want to enforce the language in here. Google should be pretty good at using the best language for the user.
```suggestion
<script src="https://www.google.com/recaptcha/api.js"></script>
```